styles.css

```diff
@@ -52,25 +54,98 @@ li {
   padding-bottom: 15%;
 }
 
-li a {
-  color: inherit;
-  text-decoration: none;
+.title {
+  cursor: pointer;
+  color: #28352f;
+  font-weight: 600;
 }
 
-.toolbar {
+/* Properties for Nav-bar */
+.navbar {
   display: flex;
+  top: 0;
+  height: 10vh;
   justify-content: space-between;
-  flex: 1;
+  align-items: center;
+  padding: 0 20px;
 }
 
-.toolbar a {
-  text-decoration: none;
+.list-item {
+  padding: 10px;
+  list-style: none;
+}
```

Comment:
- [ ] I think it would be a great idea to add some margin to the `list-item` like `margin: 0 20px;` because currently, it is closer to the edge compared to the original design. 😊